src/test/java/com/solvd/micro9/users/integration/EsProducerIT.java

```diff
@@ -0,0 +1,65 @@
+package com.solvd.micro9.users.integration;
+
+import com.google.gson.Gson;
+import com.solvd.micro9.users.domain.aggregate.User;
+import com.solvd.micro9.users.domain.es.Es;
+import com.solvd.micro9.users.domain.es.EsStatus;
+import com.solvd.micro9.users.domain.es.EsType;
+import com.solvd.micro9.users.domain.es.EsUser;
+import com.solvd.micro9.users.messaging.EsProducer;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.UUID;
+
+@Slf4j
+@SpringBootTest
+@DirtiesContext
+public class EsProducerIT extends TestcontainersTest {
```

Comment:
@eyakauleva traditionally, test classes and methods have package-private modifier

Reply:
fixed 1219506